tools/jasmine/bootstrap.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {cleanTestTmpDir} from '../../ng-dev/utils/testing/bazel-env';
+
+/**
+ *  The `boot` utility function from `@bazel/jasmine` allows initialization before the runner, with
+ *  protections to prevent calling `boot` again.
+ */
+const {boot} = require('@bazel/jasmine');
+
+// Initialize jasmine, because beforeEach and other jasmine "globals" are not defined in the
+// environment until jasmine has been initialized.  Since our bootstrap explicitly is meant to use
+// these globals, we need to ensure that jasmine has already initialized before the major content
+// of our script executes.
+boot();
```

Comment:
What happens if boot is called a second time? just checking to make sure we don't run tests twice.

Reply:
Asking because I think, to really test that it only runs once, we'd need to move the testCount constant out of the closure to the file toplevel, right?

Reply:
I can't think of how it would get called again, but I added in an `undefined` check to be sure.